keyboards/handwired/onekey/keymaps/encodertest/keymap.json

```
@@ -0,0 +1,18 @@
+{
+  "keyboard": "handwired/onekey/pytest",
+  "keymap": "encodertest",
+  "layout": "LAYOUT",
+  "layers": [
+    [
+      "KC_ENTER"
+    ]
+  ],
+  "author": "",
+  "notes": "",
+  "encoders": [
+    {
+      "clockwise": "KC_UP",
+      "counter": "KC_DOWN"
+    }
```

I think perhaps we should attempt to allow different encoders on a per-layer basis to match the keymap, before anything proliferates with "locking in" only a singular function of the encoder as per the current implementation.

```suggestion
    [
      {
        "clockwise": "KC_UP",
        "counter": "KC_DOWN"
      }
    ]
```

---

I wanted to go with the simplest implementation possible for now; those that really want full encoder support likely won't be using this CLI anyway.